src/index.ts

```diff
@@ -1,86 +1,94 @@
-import { createConnection } from "typeorm";
+import "reflect-metadata"; // this shim is required
+import { useExpressServer } from "routing-controllers";
+import connection from "./connection";
+import express = require("express");
+import helmet = require("helmet");
```

Comment:
This doesn't look like valid JS? I noticed that VSCode is a bit dodgy in the refactoring department.

Reply:
it's a typescript compromise, giving you something that's imported without defining a global constant.

I'm a fan because it allows for `import` to be a marker of, well, imports :) Thoughts?

Reply:
Ohhh, so VSCode actually was right and I was wrong. :D

Works for me!